student.rb

```diff
@@ -6,6 +6,7 @@ class Student < Person
 
   def initialize(age, classroom, parent_permission: true, name: 'Unknown')
     super(name, age, parent_permission)
+
     @classroom = classroom
   end
 
```

Comment:
![image](https://github.com/naanmohammed/oop_school_library/assets/93039359/ef124138-b6b6-4bc6-a814-cfcf735a7da6)

- [ ] It would be nice if you fix this behavior of your app because when I try to create a student then it shows the error which you can see in the image above 🆙 Kindly fix this error to make your app work as expected 😊